### PR TITLE
Rubicon adapter: accept accountId, siteId, zoneId as strings

### PR DESCRIPTION
**Pull request by And1sS**
Changes from https://github.com/prebid/prebid-server-java/pull/1627

### CODE REVIEW

File: openrtb_ext/imp_rubicon.go [hunk 1]
```diff
@@ -6,9 +6,9 @@ import (
 
 // ExtImpRubicon defines the contract for bidrequest.imp[i].ext.rubicon
 type ExtImpRubicon struct {
-	AccountId int                `json:"accountId"`
-	SiteId    int                `json:"siteId"`
-	ZoneId    int                `json:"zoneId"`
+	AccountId json.RawMessage    `json:"accountId"`
+	SiteId    json.RawMessage    `json:"siteId"`
+	ZoneId    json.RawMessage    `json:"zoneId"`
```
Comment (VeronikaSolovei9) [inline openrtb_ext/imp_rubicon.go:11]: Would it be easier to use `json.Number` instead? Please refer to this PR: https://github.com/prebid/prebid-server/pull/2105/files

Comment (And1sS) [reply openrtb_ext/imp_rubicon.go]: Yeah, thanks for suggestion!

Comment (VeronikaSolovei9) [reply openrtb_ext/imp_rubicon.go]: Thank you for changing the code! 
May I ask you to add json test where imp.ext.{accountId/siteId/zoneId} have string values? Similar to what you already have in `exemplary`, but with strings instead?

**Review by VeronikaSolovei9 — APPROVED** [review]
LGTM

File: adapters/rubicon/rubicon.go [hunk 2]
```diff
@@ -408,9 +408,21 @@ func (a *RubiconAdapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *ada
 			continue
 		}
 
+		siteId, err := rubiconExt.SiteId.Int64()
+		if err != nil {
+			errs = append(errs, err)
+			continue
```
Comment (AlexBVolcy) [inline adapters/rubicon/rubicon.go:414]: This is just for my own understanding, but is there a chance that the error we can get back is bad enough that we wouldn't want to continue execution? 

I see this style of error handling in a lot of places in this file, so I was just wondering.

Comment (And1sS) [reply adapters/rubicon/rubicon.go:414]: In most cases we would like to continue execution.

**Comment by VeronikaSolovei9** [comment]
Hi guys, hope you had a good holidays! 
Please add this change to documentation repository: https://github.com/prebid/prebid.github.io/blob/master/dev-docs/bidders/rubicon.md

**Review by AlexBVolcy — APPROVED** [review]

**Comment by And1sS** [comment]
Documentation updates related pr: https://github.com/prebid/prebid.github.io/pull/3496

**Pull request merged**